lib/init.coffee

```diff
@@ -10,6 +15,82 @@ module.exports =
       type: 'string'
       default: ''
 
-
   activate: ->
-    console.log 'activate linter-erlc'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-erlc.executablePath',
+    (executablePath) =>
+      @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'Erlc'
+      grammarScopes: ['source.erlang']
+      scope: 'file'
+      lintOnFly: false
+      lint: (textEditor) =>
+        filePath = textEditor.getPath()
+        args = ["-Wall", "-v", "-o", os.tmpDir()]
+
+        pa = atom.config.get 'linter-erlc.pa'
+        for path in pa.split(",").map((x) -> x.trim()).filter((x) -> x)
+          args.push("-pa")
+          args.push(path)
+
+        projectPath = null
+        if path.basename(path.dirname(filePath)) == 'src'
+          projectPath = path.dirname(path.dirname(filePath))
+
+        includeDirs = atom.config.get 'linter-erlc.includeDirs'
+        for includePath in includeDirs.split(",").map((x) -> x.trim()).filter((x) -> x)
```

Comment:
If you change `includeDirs` into an [array](https://atom.io/docs/api/v1.4.0/Config#array), Atom will automatically split the entered text into elements for you, while stripping whitepsace from the ends. It also clearly shows your intent for that to be an array.